contrib/kube-prometheus/assets/prometheus/rules/kube-apiserver.rules

```diff
@@ -0,0 +1,38 @@
+ALERT K8SApiserverDown
+  IF up{job="apiserver"} == 0
+  FOR 15m
+  LABELS {
+    severity = "warning"
+  }
+  ANNOTATIONS {
+    summary = "API server unreachable",
+    description = "An API server could not be scraped.",
+  }
+
+# Disable for non HA kubernetes setups.
```

Comment:
Actually, seems like that should be fine for single and HA setups alike and the above alert being redundant, no?